project/AppDependencies.scala

```diff
@@ -33,14 +30,4 @@ object AppDependencies {
 
   val akkaVersion = "2.6.10"
   val akkaHttpVersion = "10.1.12"
```

Comment:
please delete it as you are not using it

Reply:
done